prowlarr/umbrel-app.yml

```
@@ -2,7 +2,7 @@ manifestVersion: 1
 id: prowlarr
 category: media
 name: Prowlarr
-version: "0.4.4-nightly-build-2"
+version: "prowlarr:1.7.4-nightly"
 tagline: Prowlarr is an indexer manager/proxy
 description: >-
   Prowlarr is an indexer manager/proxy built on the popular *arr .net/reactjs base stack to integrate with your various PVR apps. Prowlarr supports management of both Torrent Trackers and Usenet Indexers. It integrates seamlessly with Lidarr, Mylar3, Radarr, Readarr, and Sonarr offering complete management of your indexers with no per app Indexer setup required (we do it all).
```

Let's add the this line from our last release notes:

```suggestion
  Prowlarr is an indexer manager/proxy built on the popular *arr .net/reactjs base stack to integrate with your various PVR apps. Prowlarr supports management of both Torrent Trackers and Usenet Indexers. It integrates seamlessly with Lidarr, Mylar3, Radarr, Readarr, and Sonarr offering complete management of your indexers with no per app Indexer setup required (we do it all).
  
  
  On installation of Sonarr, Lidarr or Radarr, Prowlarr will automatically set up the app's connection within its settings.
```